typescript-generator-core/src/main/java/cz/habarta/typescript/generator/parser/DeprecationEnricher.java

```diff
@@ -14,21 +17,23 @@
 
 public class DeprecationEnricher {
 
+    public static final String DEPRECATED = "@deprecated";
+
     public Model enrichModel(Model model) {
         final List<BeanModel> beans = mapList(model.getBeans(), this::enrichBean);
         final List<EnumModel> enums = mapList(model.getEnums(), this::enrichEnum);
         final List<RestApplicationModel> restApplications = mapList(model.getRestApplications(), this::enrichRestApplication);
         return new Model(beans, enums, restApplications);
     }
 
-    private BeanModel enrichBean(BeanModel bean) {
+    public BeanModel enrichBean(BeanModel bean) {
```

Comment:
Made most methods public to enable direct call in custom plugin.

Reply:
These methods are internal and we should keep possibility to refactor this class and remove these methods or change their signature. Could you call just `enrichModel` method from your plugin? It should not duplicate `@deprecated` tags.

In fact I considered whole class internal and not intended it to be called from other places but I understand that it could be useful.